lib/src/scss-shared/_nx-colors.scss

```diff
@@ -77,14 +80,15 @@ $nx-text-color: #474747;
 $nx-text-color-disabled: #949494;
 $nx-text-color-error: $nx-fail-color;
 
-$nx-link-color: #045dd1;
-$nx-link-color-active: #2783fb;
-$nx-link-color-hover: #034396;
+$nx-link-color: $nx-blue-600;
+$nx-link-color-active: $nx-blue-400;
+$nx-link-color-hover: $nx-blue-800;
 $nx-link-color-disabled: #949494;
 
-$nx-background-color-interactive: #2476c3;
-$nx-background-color-interactive-active: #0046b1;
-$nx-background-color-interactive-disabled: #97cbee;
+// Basic interactive states
+$nx-selected-background: $nx-blue-200;
+$nx-active-background: $nx-blue-200;
+$nx-hover-background: #f1f3fa;
```

Comment:
Is this going to be in the teal palette?

Reply:
> Is this going to be in the teal palette?

Yes it is, I probably should have removed this, but I'm going to do the grey and then the teal. I'll remove it if you would prefer.